llvm/test/MC/Disassembler/X86/apx/bmi2.txt

```diff
@@ -0,0 +1,226 @@
+# RUN: llvm-mc --disassemble %s -triple=x86_64 | FileCheck %s --check-prefixes=ATT
+# RUN: llvm-mc --disassemble %s -triple=x86_64 -x86-asm-syntax=intel --output-asm-variant=1 | FileCheck %s --check-prefixes=INTEL
+
```

Comment:
Could you add comment like 
```
## mulx
```
when we start checking a new menonic ? This would improve the readability.

Reply:
Will do